resources/views/search.blade.php

```diff
@@ -26,7 +26,7 @@
 
                             <div class="col pl-0">
                                 <a href="{{ route('account.show', ['username' => $user->username]) }}">
-                                    <h4>{{ $user->name }} <small>{{ $user->username }}</small></h4>
+                                    <h4>{{ $user->name }}@if($user->private_profile)<i class="fas fa-user-lock"></i>@endif <small class="text-muted">{{ '@'.$user->username }}</small></h4>
```

Comment:
```suggestion
                                    <h4>{{ $user->name }}@if($user->private_profile)<i class="fas fa-user-lock"></i>@endif <small class="text-muted">@{{ $user->username }}</small></h4>
```

Reply:
Will not work. This will cause blade to stop working.